src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Resources/xlf/Strings.cs.xlf

```diff
@@ -84,7 +84,7 @@
       </trans-unit>
       <trans-unit id="ConfigurationNotSupportedMessageValue">
         <source>The specified value '{0}' for '{1}' is not supported by source-generated P/Invokes. If the specified value is required, use a regular `DllImport` instead.</source>
-        <target state="needs-review-translation">Určená hodnota {0} pro {1} se nepodporuje u zdrojem generovaných volání P/Invokes. Pokud je určená konfigurace povinná, použijte místo ní normální Dllimport.</target>
+        <target state="new">The specified value '{0}' for '{1}' is not supported by source-generated P/Invokes. If the specified value is required, use a regular `DllImport` instead.</target>
```

Comment:
Is it intentional that the previous translations are deleted and the state is changed to new? @danmoseley @ericstj who would be the right person to check? This PR was opened by the OneLocBuild automation task.

Reply:
I do not know who owns this. @ericstj would have to answer.

Reply:
I usually expect the workflow to be that developers make changes to a resx which create `new` elements in the xlf files when the developer builds their changes.  Those get checked in as part of the PR.  The daily build happens and "hands those off" to loc.  The same daily build examines the loc resources for hand-back and generates a PR like this if there are changes to hand-back.  Usually, these PRs make changes to transition elements from `state` `new` to `translated`.  I have not seen the state `needs-review-translation` before.

If I look at the history for this file, I see that it happened as a result of @elinor-fung's change https://github.com/dotnet/runtime/pull/67863

This changed a string which was already translated, so perhaps that's what made the state change.  I think this looks normal and we should merge it and then just let the translators re-translate the strings.  @jonfortescue does that sound right?

Further docs on this can be found 
https://github.com/dotnet/arcade/blob/main/Documentation/OneLocBuild.md
and the links at the top of this PR.

Reply:
OneLocBuild makes string changes marked as new. You can merge this and then expect translations to flow within the translation SLA (1 week from the string being introduced/changed).

Reply:
so yes, you're correct @ericstj